LEGO1/legoanimactor.cpp

```diff
@@ -0,0 +1,24 @@
+#include "legoanimactor.h"
+
+// 0x100f057c
+static char* g_legoAnimActorClassName = "LegoAnimActor";
+
+// OFFSET: LEGO1 0x1000fb90 STUB
+const char* GetClassNameInternal()
+{
+  return g_legoAnimActorClassName;
+}
+
+// OFFSET: LEGO1 0x1000fb90 STUB
+const char *LegoAnimActor::GetClassName() const
+{
+  return GetClassNameInternal();
+}
```

Comment:
These are caused by multiple inheritance, and automatically generated by the compiler. I think.

Reply:
The real method is at `0x1000fba0`.